src/script/script.h

```diff
@@ -412,6 +412,9 @@ class CScript : public CScriptBase
     CScript(std::vector<unsigned char>::const_iterator pbegin, std::vector<unsigned char>::const_iterator pend) : CScriptBase(pbegin, pend) { }
     CScript(const unsigned char* pbegin, const unsigned char* pend) : CScriptBase(pbegin, pend) { }
 
+    // Define a copy constructor so that operator= is a copy instead of a move to avoid undefined behavior
```

Comment:
If this is needed, there is another problem. Why does a move cause UB? And why is a move being invoked in the first place with operator=?

Reply:
No idea. Maybe @practicalswift can shed some light here?

Reply:
See also: https://github.com/bitcoin/bitcoin/pull/17156#issuecomment-544160293